lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -3,8 +3,12 @@
         <div class="course-content">
             <div class="course-details">
                 <h5 class="course-title">
-                    <% if ( (marketing_url || course_url) &&  !is_unfulfilled_entitlement) { %>
-                        <a href="<%- marketing_url || course_url %>" class="course-title-link">
```

Comment:
We can simplify this by moving the conditional logic and storing the url in a variable.  Then only having the HTML in the template once.  Removing duplication.